packages/ui/ViewManager/tests/ViewManager-specs.js

```diff
@@ -203,4 +202,58 @@ describe('ViewManager', () => {
 		expect(actual).toBe(expected);
 		done();
 	});
+
+	test('should update the View reverseTransition prop.', done => {
+		const subject = mount(
+			<ViewManager noAnimation index={0} duration={0}>
```

Comment:
1. Testing `reverseTransition` without the `arranger` has no effect. Please update accordingly.
2. Do you think `noAnimation` makes difference as well?

Reply:
It has effect. It updates the `reverseTransition` prop of View component and this test is checking View's `reverseTransition`. This test is for testing if ViewManager's `getDerivedStateFromProps` method is setting its own `reverseTransition` state according to its new props and pass it to View. `noAnimation` just followed from copy and paste from the last test, this has nothing to with this test. It can be removed or left as it is. 
Lmk if I'm not understanding how arranger work in ViewManager.

Reply:
I should've put the comment in a different line to make it clear.

From `View`'s perspective, `reverseTransition` prop is only meaningful when `ViewManager` has `arranger` prop declared.  If you look at how `prepareTransition()` is called in `View`, `reverseTransition` is respected only if there's `arranger` prop. And similarly, `noAnimation` will disregard `reverseTransition` in `View` as well. This test verifies whether the prop is passed down correctly to its children, and this is fine. But this would never be the normal usage. I'm on the fence that our test cases should cover the actual usages. WDYT?

Reply:
I added this test for add unit tests for `reverseTransition` props?" and to test if this lifecycle change is breaking the original logic or not. I think you want to add some other test cases.

Reply:
Perhaps? This could be in the scope of `View` rather than `ViewManager` depending on how you look at it. But I'm arguing that `reverseTransition` won't have any impact on how the component behaves without these props. Why does it matter if it passes down to children when it doesn't consume?